### DISCUSSION TITLE
Porting Coin3D to WEB with WebAssembly

### DISCUSSION THREAD

**Discussion by kaankoroglu** [coin3d/coin#515] [Ideas]
I am looking for a open inventor viewer in WEB environment and searched the internet but could not find any alternative. So my question is, is that possible to compile Coin3D libraries with the Emscripten so that we can use it on the WEB environment.

**Comment by famorcia** [comment]
Hello, 
have you tried compilation with Emscripten? 
Which issues have you found?
Maybe this could be material for discussion.

**Reply by kaankoroglu** [reply]
I have managed to compile coin3d it with Emscripten. Right now I have started to write a wrapper something like SoQt to able use it within the browser.

**Reply by tritao** [reply]
@kaankoroglu Are you still working on it?
